docs/docs/configuration/filtering.md

```diff
@@ -202,11 +228,49 @@ See https://avd.aquasec.com/misconfig/avd-aws-0081
 ```
 </details>
 
-## By Finding IDs
+## Suppression
+You can filter the results by
+
+- Finding IDs
+- Rego
+- Vulnerability Exploitability Exchange (VEX)
```

Comment:
same

Reply:
Thanks for your feedback. Added.
https://github.com/aquasecurity/trivy/pull/6084/commits/9ac010c8470ee637d3330a13ed2c14bb1175e240